parser/parser_test.go

```diff
@@ -582,8 +583,21 @@ func TestFromFile(t *testing.T) {
 		}, {
 			"./testdata/workflows/dataInputSchemaValidation.yaml", func(t *testing.T, w *model.Workflow) {
 				assert.NotNil(t, w.DataInputSchema)
-
-				assert.Equal(t, "sample schema", w.DataInputSchema.Schema)
+				expected := model.FromString("sample schema")
```

Comment:
Add a test to load a external resource. Ex.: `file://testdata/datainputschema.json`